docs/myriad-configuration.md

```diff
@@ -63,10 +69,14 @@ executor:
   jvmMaxMemoryMB: 256   # Xmx for myriad's executor that launches Node Manager.
   path: file://localhost/usr/local/libexec/mesos/myriad-executor-0.0.1.jar  # Path for the myriad's executor binary.
                                                                             # Also supports, hdfs:// notation.
+  #These are for remote distribution
+  #path: hadoop-2.5.0/share/hadoop/yarn/lib/myriad-executor-0.0.1.jar #this should be relative if nodeManagerUri is set
+  #nodeManagerUri: hdfs://namenode:port/dist/hadoop-2.5.0.tar.gz # the uri to d/l hadoop from
```

Comment:
I'm confused about the relationship between executor.path and executor.nodeManagerUri. Are you assuming that the executor is bundled into the tarball from the nodeManagerUri? We shouldn't have to bundle the myriad executor with the NM hadoop tarball. At some point in the future, we may want multiple executor URIs: one for the hadoop binary, one for the executor binary, one for hadoop config, and maybe one for preloaded [meta]data or other custom files. How would we want to express that?